DOCSP-35827: Update SetComment() parameter type (#340)
mongodb/docs-golang: source/whats-new.txt

   :ref:`Connection Example Code <go-connection-example-code>` in the
   Connection Guide.

The 2.0 {+driver-short+} release includes the following improvements and fixes:

> - For all options structs, the ``Comment`` field has a value of type ``any`` instead of type
>   ``string``. The ``SetComment()`` method takes a parameter of type ``any``.

Reviewer: ```suggestion
- The ``Comment`` field in any options struct takes a value of type ``any`` instead of type
  ``string``. To set this field, you can use the ``SetComment()`` method and pass a parameter of type ``any``.
```
